### PR TITLE
Port to jbuilder

### PR DESCRIPTION
**Pull request by mseri**
Will require a spec file update (ready and tested)

### CODE REVIEW

**Comment by mseri** [comment]
The tests are currently broken (and failing). Interesting enough, it is not due to this PR: they are already broken in master.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+OPAM_PREFIX?=$(DESTDIR)$(shell opam config var prefix)
+OPAM_LIBDIR?=$(DESTDIR)$(shell opam config var lib)
+
+.PHONY: release build install uninstall clean test doc reindent
```
Comment (gaborigloi) [inline Makefile]: +gh-pages

Comment (mseri) [reply Makefile]: That was there by mistake, well spotted. Thanks!

File: xapi-forkexecd.opam [hunk 2]
```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/"
+bug-reports: "https://github.com/xapi-project/forkexecd/issues"
+dev-repo: "git://github.com/xapi-project/forkexecd"
+tags: [ "org:xapi-project" ]
+
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: ["jbuilder" "runtest" "-p" name]
```
Comment (gaborigloi) [inline xapi-forkexecd.opam]: This also takes the number of jobs: `"-j" jobs` -  see https://jbuilder.readthedocs.io/en/latest/usage.html#tests

Comment (mseri) [reply xapi-forkexecd.opam]: There is only a test that need to run in a single process

Comment (gaborigloi) [reply xapi-forkexecd.opam]: But maybe this is also for the compilation of the tests?

**Comment by mseri** [comment]
They are both going to xs-extras, they are in the same specfile

File: xapi-forkexecd.opam (hunk 2 above)
Comment (gaborigloi) [reply xapi-forkexecd.opam]: Or only for the compilation of the tests? 🤔

Comment (mseri) [reply xapi-forkexecd.opam]: No idea, probably for the compilation, but it's not clear from the documentation. Anyways, the test is a single file as well, so multiple jobs would not make any difference.

**Pull request merged**